test/chart/LineChart.spec.tsx

```diff
@@ -751,7 +750,7 @@ describe('<LineChart /> - Rendering two line charts with syncId', () => {
     expect(container.querySelectorAll('.recharts-tooltip-cursor')).toHaveLength(2);
 
     // make sure tooltips display the correct values, synced by data value
-    expect(screen.queryByText('400')).toBeTruthy();
+    expect(screen.queryByText('300')).toBeTruthy();
```

Comment:
This is surprising. Did the test previously capture behaviour that was a bug itself? 
Could you please add a storybook test (i.e. in a play function for a story with syncId) to verify that the syncId still works correctly?

Reply:
Great, I'll give it a try.